content/bibs/pubs.bib

```diff
@@ -1338,15 +1338,49 @@ @article{Zheng2016b
 
 
 
+@article{shen2020high,
+  title={High-dimensional independence testing and maximum marginal correlation},
+  author={Shen, Cencheng},
+  author+an={},
+  url={https://arxiv.org/abs/2001.01095},
+  journal={arXiv},
+  year={2020},
+  keywords={in-review}
+}
 
+@article{lawrence2021standardizing,
+  title={Standardizing human brain parcellations},
+  author={Lawrence, Ross M and Bridgeford, Eric W and Myers, Patrick E and Arvapalli, Ganesh C and Ramachandran, Sandhya C and Pisner, Derek A and Frank, Paige F and Lemmer, Allison D and Nikolaidis, Aki and Vogelstein, Joshua T},
+  author+an={1=trainee, 2=trainee, 10=highlight},
+  journal={Scientific data},
+  url={https://www.nature.com/articles/s41597-021-00849-3},
+  volume={8},
+  number={1},
+  pages={1--9},
+  year={2021},
+  publisher={Nature Publishing Group},
+  keywords = {peer-reviewed}
+}
 
+@article{powell2021ten,
```

Comment:
I think you missed this one~

Reply:
Turns out there have been not just duplicates but triplicates perhaps even quad that could be found either by title and their codes. This particular one is the duplicate by code, and I deleted this one